docker-compose.yml

```diff
@@ -74,8 +74,8 @@ services:
       - AWS_ACCESS_KEY_ID
       - AWS_SECRET_ACCESS_KEY
       - SAMPLES_BUCKET_NAME=idseq-samples-development
-      - IDSEQ_AIRBRAKE_PROJECT_ID
-      - IDSEQ_AIRBRAKE_PROJECT_KEY
+      - AIRBRAKE_PROJECT_ID
+      - AIRBRAKE_PROJECT_KEY
```

Comment:
weird, how has this been working before?

Reply:
I'd guess that in local dev, Airbrake doesn't get used.